### ISSUE TITLE
System messages and pop up window of unknown user

### ISSUE THREAD

**Issue by michande** [keybase/client#3854]
Keybase GUI Version: 1.0.16-20160809160037+24badd1

Every time I open a folder I get a system message that en_us is not a user and then the folder correctly opens but also a user gui screen opens for a unknown user that has chosen the handle en.

My guess is, well nonsensical, but nevertheless. en_us looks like some locale but why it then would cut out _us when looking for a user I don't follow and the user doesn't follow me either is quite the tricky one.

my log id: 9b8fce2f8c5c4d1ee893031c

**Comment by cjb** [comment]
There's probably something in your environment (e.g. shell config) that looks for a file called en_us in every directory you cd into.

**Comment by strib** [comment]
Yes, and the leftover `en` is likely some autocomplete configuration in your shell -- it tries to look up every prefix of `en_us` and `en` happens to be a Keybase user.  We have a PR out that will stop creating directories in that case but it's not merged yet.

**Comment by michande** [comment]
Yes, seems like something like that. Didn't test it in terminal though, but will do and see if it's just a GUI problem. After all the pop ups in the gui and system tray it cd to the correct folder anyhow so functional but well glitchy.

**Comment by Perturbee** [comment]
I have the exact same thing happening when using the GUI (run_keybase) on KDE (Debian) and clicking the "open" link on my own public folder on the GUI. Apparently that parses en_us along to Dolphin as well as suggesting to track user "en".

keybase version 1.0.18-20170103235713+b2b347b

I also did exactly the same on xcfe4 (linuxmint7) which opened my folder in Nemo without any problems. So it looks like it's a specific KDE/Dolphin issue.

![en_us_not_a_user_notification](https://cloud.githubusercontent.com/assets/4726251/21723512/003c9450-d431-11e6-9b00-d032a4929232.jpg)
